src/System.Data.SqlClient/src/System/Data/SqlClient/SqlCommand.cs

```diff
@@ -20,15 +20,82 @@ namespace System.Data.SqlClient
 {
     public sealed partial class SqlCommand : DbCommand, ICloneable
     {
+        private sealed class ExecuteNonQueryAsyncContext
+        {
+            public SqlCommand Command;
```

Comment:
Can you change these public to internal, so that I dont have to scroll to the class definition to conclude the final visibility of these types and functions ?